From a5e5579e1071f5bf0519479f6a43d53c5b8afde4 Mon Sep 17 00:00:00 2001
From: Zhen Lei <thunder.leizhen@huawei.com>
Date: Tue, 18 Jul 2023 16:05:41 +0000
Subject: [PATCH 34/36] [Huawei] rcu: Restore RCU_CPU_STALL_CPUTIME to be
 disabled by default

Offering: HULK
hulk inclusion
category: bugfix
bugzilla: 188933

--------------------------------

RCU_CPU_STALL_CPUTIME was set to be enabled by default when backport
commit be42f00b73a0 ("rcu: Add RCU stall diagnosis information"), and now
it is restored to be disabled by default, consistent with the community.

Fixes: a11a550323ac ("rcu: Add RCU stall diagnosis information")
Signed-off-by: Zhen Lei <thunder.leizhen@huawei.com>
---
 kernel/rcu/Kconfig.debug | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kernel/rcu/Kconfig.debug b/kernel/rcu/Kconfig.debug
index dfaef1b50424..65103e9fa6f8 100644
--- a/kernel/rcu/Kconfig.debug
+++ b/kernel/rcu/Kconfig.debug
@@ -94,7 +94,7 @@ config RCU_CPU_STALL_TIMEOUT
 config RCU_CPU_STALL_CPUTIME
 	bool "Provide additional RCU stall debug information"
 	depends on RCU_STALL_COMMON
-	default y
+	default n
 	help
 	  Collect statistics during the sampling period, such as the number of
 	  (hard interrupts, soft interrupts, task switches) and the cputime of
-- 
2.17.1

